tweets/<your-path>.tweet

```diff
@@ -0,0 +1,2 @@
+Wanted to note a time change for folks interested in exploring bringing a project to Operate First 
+this Wednesday, it has moved out 1/2 hour to 8:30 US Eastern time....Folks wanting an invite ping @bburns_rh
```

Comment:
```suggestion
Interested in exploring bringing your project to Operate First ? Ping @bburns_rh and join our Project Coordination Session this Wednesday at 8:30 US Eastern time. 
```
@billburnseh since we have not made a tweet in the past around the previous time, I would suggest just putting it out as a call for projects rather than notification for change. WDYT?

I think some trailing whitespaces are causing pre-commit to fail.

Small nit - but could you please change the filename to any preferred tweet name and add it under the monthly directory as per the instructions [here](https://github.com/operate-first/operate-first-twitter#how-to-tweet)?